src/main/java/me/A5H73Y/parkour/utilities/Utils.java

```diff
@@ -912,9 +896,11 @@ public static void toggleVisibility(Player player, boolean override) {
     }
 
     public static void forceVisible(Player player) {
-        for (Player players : Bukkit.getOnlinePlayers()) {
-            players.showPlayer(player);
+        for (Player p : Bukkit.getOnlinePlayers()) {
+            if (Static.containsHidden(p.getName())) continue;
```

Comment:
Similarly this is bad practice, please use curly brackets. Also, just invert the condition and it's the same outcome but tidier.